docs/topics/wip_plugins.md

```diff
@@ -1,23 +1,21 @@
 # Introduction
 
-As your bot grows you will probably want to split the code into multiple files to improve readability and extendability.
+As your bot grows you'll probably want to split code into multiple files to improve readability and extendability.
 
 Hata offers you a solution called plugin loading.
 
-As it comes from the name, plugins can be plugged in and out, allowing your application to be more modular.
-Any added, removed event handler or interaction is tracked while loading a plugin defining what changes should be
-reversed if required.
+Plugins can be plugged in and out, allowing your application to be more modular.
 
-The main aim of hata's plugin loader is to preserve traditional code flow inside of plugins without introducing weird
+The main aim of Hata's plugin loader is to preserve traditional code flow inside of plugins without introducing weird
 top level classes.
-When designing it an important factor was to not limit it's support for a single client instance producing a pretty
-unique solution for the problem. The concept is based on presetting variables before the module is executing, but more
-about this later.
+
+When designing this system it was an important factor to not limit its support for a single client instance.
```

Comment:
Sounds strange, maybe a bit of restructuring like

"Support for single client applications was a significant factor under consideration while designing this system"

Reply:
Will copy paste